src/org/zaproxy/zap/extension/websocket/manualsend/ManualWebSocketSendEditorDialog.java

```diff
@@ -212,15 +232,22 @@ public WebSocketMessagePanel(ChannelSortedListModel channelsModel, JToolBar cont
 			controlToolbar.add(wsUiHelper.getChannelLabel());
 			controlToolbar.add(wsUiHelper.getChannelSingleSelect());
 			wsUiHelper.getChannelSingleSelect().setSelectedIndex(0);
-			
+			addWebSocketReopenListener(sender);
+
+			JButton btnReopen = getBtnReopen(sender);
+			JButton btnReopenEdit = getBtnReopenEdit(sender);
+			controlToolbar.add(btnReopen);
+			controlToolbar.add(btnReopenEdit);
```

Comment:
Instead of doing this in two steps you could just `controlToolbar.add(getBtnReopen(sender));`?